master/usage/ip-pool-manipulation.md

```diff
@@ -0,0 +1,82 @@
+---
+title: IP Pool Manipulation
+---
+The 
+[IP Pools]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/ippool)
+that Calico starts up with can be manipulated and changed.  This document
+describes how to do that using the `calicoctl` utility.
+
+## Retrieving IP Pool information
+
+The current IP Pools can be retrieved with the following commmand.
+
+```shell
+calicoctl get ipPool
+```
+
+This will return information similar to the following.
+
+```shell
+CIDR
+192.168.0.0/16
+
+```
+
+To retrieve the information about a specific IP Pool you must specify the pool
+also.  The `-o yaml` will output in a format that can be captured, manipulated,
+and then used to apply the desired changes.
+
+```shell
+calicoctl get ipPool 192.168.0.0/16 -o yaml
+```
+
+## Manipulating an existing pool
```

Comment:
This guide is good, but I want the focus of this guide to be less about standard `get`, `edit`, `apply`, `delete`, and more of a step by step guide of "How to Enable IPIP and NAT-Outgoing on a pool". Can you change this^ title to reflect that?